docs/source/stability_policy.rst

```diff
@@ -0,0 +1,132 @@
+Stability Policy
+================
+
+The ``python-telegram-bot`` library (PTB) tries to provides a stable interface to its users.
+As both the user base and the complexity of the library has grown significantly since PTBs conception, we introduced below explicit stability Policy with version NEXT.VERSION.
```

Comment:
I like the sentence, I do not like it in there, because I wouldn't really care why the policy exists when reading the policy itself, especially in future time. Put it in the announcement or in the readme or wherever we link to it. If you want to keep the version reference, maybe

```suggestion
Starting with version NEXT.VERSION, PTB gives itself/its contributors the following stability policy
```

Id probably move it to the end of the paragraph however.

Reply:
Good points 👍
I will keep a version reference as it's important to me that users can see which versions this applies to and which not